### ISSUE TITLE
parse_ntfs() offline $MFT file parsing

### ISSUE THREAD

**Issue by ablescia** [Velocidex/velociraptor#3519]
Can I use the parse_ntfs function to parse a standalone $MFT file?

I'm participating in a CTF that gave me a single $MFT file that I need to analyze to complete the challenge.

Using the following VQL code, I can retrieve basic NTFS file information:

```
SELECT * FROM parse_mft(filename="C:/analysis/$MFT", accessor="ntfs")
```
It works! I see the $MFT file entries!

After that, I tried to use the parse_ntfs function to access a specific inode using the following code:

```
SELECT parse_ntfs(
    device='"C:/analysis/$MFT',
    inode="17551-128-0")
FROM scope()
```

However, I see that this VQL code parses the **C:/$MFT** file of the host used to analyze the file.How can I solve this issue? Thanks

**Comment by scudette** [comment]
This is currently not supported but maybe we should add this option

**Comment by scudette** [comment]
Fixed in 0.72.4
